src/sentry/templatetags/sentry_assets.py

```diff
@@ -133,3 +133,12 @@ def _unwrap_content(self, text):
         if matches:
             return matches.group(1).strip()
         return text
+
+
+@register.simple_tag
+def organization_avatar(organization):
+    avatar = organization.avatar.first()
```

Comment:
I'm generally suspicious of .first() as it applies an order which in most cases doesn't work well for us. However, this table is on the small side and the query will hit an index.

Reply:
I can also use `.last()`. This is what I've seen was used for getting the avatar (which feels weird to me tbh).

Reply:
Can we address this and get this merged?

Reply:
>Can we address this and get this merged?

Don't see anything to address here. This is how we use avatars everywhere unfortunately.